### PR TITLE
Make Image Pane scrolling browser independent

### PR DESCRIPTION
**Pull request by dzimmerer**
Instead of the browser-dependent scroll wheel value, only use the sign value, and allow to side scroll image with Alt+Shift.

## Description
Use only the sign of the scrolling value/speed to always get a fixed scrolling offset.
Also added, if Alt+Shift is pressed to allow scrolling image in X direction instead of Y.

## Motivation and Context
https://stackoverflow.com/questions/5527601/normalizing-mousewheel-speed-across-browsers
Mouse wheel speed is different for different browsers, leading to unusable zooming and scrolling values in different browsers.

## How Has This Been Tested?
Tested on Chrome + Firefox (on ubuntu).

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] For JavaScript changes, I have re-generated the minified JavaScript code.

### CODE REVIEW

File: js/ImagePane.js [hunk 1]
```diff
@@ -61,23 +61,35 @@ class ImagePane extends React.Component {
   }
 
   handleZoom = (ev) => {
+
```
Comment (JackUrb) [inline js/ImagePane.js]: nit: remove

File: js/ImagePane.js [hunk 2]
```diff
@@ -61,23 +61,35 @@ class ImagePane extends React.Component {
...
+      else {
+        this.setState({tx: this.state.tx + scrollDirectionX * direction*50});
+        this.setState({ty: this.state.ty + scrollDirectionY * direction*50});
+      }
       ev.stopPropagation();
       ev.preventDefault();
     } else if (ev.ctrlKey) {
       // get the x and y offset of the pane
       var rect = this._paneRef._windowRef.children[1].getBoundingClientRect();
+      // Get browser independendgit scaling factor
```
Comment (JackUrb) [inline js/ImagePane.js]: nit: spelling

**Comment by JackUrb** [comment]
Hi @dzimmerer - thanks for the fix, it looks great! One question I have about the shift implementation is that Shift->scroll in most browsers is registered as horizontal scrolling already, so isn't the special case redundant?

**Comment by dzimmerer** [comment]
I thought so as well, but at least in the cases I tried the horizontal scrolling worked on the whole window/grid but not on the image pane.

**Pull request closed without merging**